src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(calculateStatistics(fromFileName), toFileName);
+
+    }
+
+    public int [] calculateStatistics(String fromFileName) {
+        File file = new File(fromFileName);
+        int [] result = new int[3];
```

Comment:
The array `result` is not self-explanatory. Consider using a class or a more complex data structure with descriptive field names to hold the statistics data.